roles/analysis/tasks/analysis-c2r.yml

```diff
@@ -15,6 +15,7 @@
   async: "{{ analysis_async_timeout_maximum | int }}"
   poll: "{{ analysis_async_poll_interval | int }}"
   failed_when: "'Pre-conversion analysis report' not in convert2rhel.stdout"
+  notify: Remove temporary configuration file
```

Comment:
Did you try it? I didn't I am just wondering what is the purpose of the `changed_when` in the other notify

```
    changed_when: true
    notify: Pre-conversion analysis report is done
```

I have a feeling that notify will work only if the task is changing the system status, and I have no idea if that is happening with shell command.

However it looks like great change!

Reply:
perfect then :)